modules/javafx.graphics/src/main/native-prism-es2/GLDrawable.c

```diff
@@ -40,27 +40,3 @@ void initializeDrawableInfo(DrawableInfo *dInfo)
     // Initialize structure to all zeros
     memset(dInfo, 0, sizeof(DrawableInfo));
 }
-
-void deleteDrawableInfo(DrawableInfo *dInfo)
```

Comment:
Maybe delete this whole file, as its only content is now a function that just calls `memset()`.

Reply:
Agreed. deleted it.